src/components/graphqlFormGenerator/components/BroadcastSetting.vue

```diff
@@ -83,16 +83,16 @@ export default {
       let ptr = object_
       let ret = ''
       while (ptr) {
-        for (const key in ptr) {
+        const keys = [...ptr.keys()]
+        if (keys && keys.length !== 0) {
+          const key = keys[0]
```

Comment:
I managed to get this traceback opening the "mutations" view which I think is related to this:

> Error in render: "TypeError: ptr.keys is not a function. (In 'ptr.keys()', 'ptr.keys' is undefined)"

Reply:
Argh. Object.keys. good catch, I forgot to test mutations.

Will fix it and see if I can write a short test to look for this error too.

Reply:
Fixed!